src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <p>
+        {' Posted by  '}
+        <UserInfo user={post.user} />
```

Comment:
`post.user` theoretically might be `undefined` - we should handle this case (to avoid error when trying to read properties from, `user` inside the `UserInfo` component)

Do not render this component if the `user` is not defined: [conditional rendering](https://react.dev/learn/conditional-rendering#logical-and-operator-)